DesktopEdge/Views/Screens/IdentityDetails.xaml.cs

```diff
@@ -110,13 +110,11 @@ public ZitiIdentity Identity {
         public MenuIdentityItem SelectedIdentityMenu { get; set; }
 
         private void Window_MouseDown(object sender, MouseButtonEventArgs e) {
-            if (e.ChangedButton == MouseButton.Left) {
-                _isAttached = false;
-                OnDetach(e);
-            }
+            if (e.ChangedButton == MouseButton.Left) IsAttached = false;

```

Comment:
The same code block for a different scope that allows left and right click attach/detachment of the window.